src/Symfony/Component/Messenger/MessageBusInterface.php

```diff
@@ -23,7 +23,7 @@
      *
      * The bus can return a value coming from handlers, but is not required to do so.
      *
-     * @param object $message
+     * @param object|Envelope $message The message or the message pre-wrapped in an envelope
```

Comment:
kinda an implementation detail.. no?

Reply:
No. That indicates you can dispatch a configured message and the bus has to handle it, not just the message itself.

Reply:
It eventually depends on the implementation :) the pure API, IMHO, is `dispatch(object $message)`.